challenges/classes.js

```diff
@@ -1,7 +1,61 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
+/* ===== Prototype Practice ===== */
+
+// Task: You are to build a cuboid maker that can return values for a cuboid's volume or surface area. Cuboids are similar to cubes but do not have even sides. 
```

Comment:
try to imitate the instructions given during sprint challenges as comments for your mon-thursday code. this way you improe the way you comment on things.